conf/config-default.yaml

```diff
@@ -14,53 +14,48 @@
-  #   - ip: 127.0.0.2               # Specific IP, If not set, the default value is `0.0.0.0`.
-  #     port: 9082
-  #     enable_http2: true
+    # - port: 9081
+    #   enable_http2: true     # If not set, default to `false`.
+    # - ip: 127.0.0.2          # If not set, default to `0.0.0.0`/
+    #   port: 9082
+    #   enable_http2: true
   enable_admin: true
-  enable_dev_mode: false            # Sets nginx worker_processes to 1 if set to true
-  enable_reuseport: true            # Enable nginx SO_REUSEPORT switch if set to true.
-  show_upstream_status_in_response_header: false # when true all upstream status write to `X-APISIX-Upstream-Status` otherwise only 5xx code
+  enable_dev_mode: false       # If true, nginx worker_processes will be set to 1.
+  enable_reuseport: true       # If true, nginx SO_REUSEPORT option will be enabled.
+  show_upstream_status_in_response_header: false  # If true, include the upstream HTTP status code in
+                                                  # the response header `X-APISIX-Upstream-Status`.
+                                                  # If false, show `X-APISIX-Upstream-Status` only if
+                                                  # the upstream response code is 5xx.
   enable_ipv6: true
 
-  #proxy_protocol:                  # Proxy Protocol configuration
-  #  listen_http_port: 9181         # The port with proxy protocol for http, it differs from node_listen and admin_listen.
-                                    # This port can only receive http request with proxy protocol, but node_listen & admin_listen
-                                    # can only receive http request. If you enable proxy protocol, you must use this port to
-                                    # receive http request with proxy protocol
-  #  listen_https_port: 9182        # The port with proxy protocol for https
-  #  enable_tcp_pp: true            # Enable the proxy protocol for tcp proxy, it works for stream_proxy.tcp option
-  #  enable_tcp_pp_to_upstream: true # Enables the proxy protocol to the upstream server
-  enable_server_tokens: true        # Whether the APISIX version number should be shown in Server header.
-                                    # It's enabled by default.
-
-  # configurations to load third party code and/or override the builtin one.
-  extra_lua_path: ""                # extend lua_package_path to load third party code
-  extra_lua_cpath: ""               # extend lua_package_cpath to load third party code
-  #lua_module_hook: "my_project.my_hook"  # the hook module which will be used to inject third party code into APISIX
-
-  proxy_cache:                      # Proxy Caching configuration
-    cache_ttl: 10s                  # The default caching time in disk if the upstream does not specify the cache time
-    zones:                          # The parameters of a cache
-      - name: disk_cache_one        # The name of the cache, administrator can specify
```

Comment:
Need to understand what it means by "administrator can specify which cache to use by name in the Admin API". Don't remember anything about cache in Admin API.